src/patternfly/components/AppLauncher/app-launcher.scss

```diff
@@ -11,7 +11,15 @@
   --pf-c-app-launcher__menu--ZIndex: var(--pf-global--ZIndex--xs);
```

Comment:
Looks like we need to add a normal `font-weight` to `.pf-c-app-launcher__menu-item`. There is a global style for `a` elements that uses a `semi-bold` font-weight. You can mimic the dropdown component for this change.